coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/utils/CoralToTrinoSqlCallTransformersUtil.java

```diff
@@ -0,0 +1,206 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.trino.rel2trino.utils;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import com.google.common.collect.ImmutableList;
+
+import org.apache.calcite.sql.SqlOperator;
+import org.apache.calcite.sql.fun.SqlStdOperatorTable;
+
+import com.linkedin.coral.com.google.common.base.CaseFormat;
+import com.linkedin.coral.com.google.common.base.Converter;
+import com.linkedin.coral.com.google.common.collect.ImmutableMultimap;
+import com.linkedin.coral.common.functions.Function;
+import com.linkedin.coral.common.transformers.OperatorBasedSqlCallTransformer;
+import com.linkedin.coral.common.transformers.SqlCallTransformer;
+import com.linkedin.coral.common.transformers.SqlCallTransformers;
+import com.linkedin.coral.hive.hive2rel.functions.HiveRLikeOperator;
+import com.linkedin.coral.hive.hive2rel.functions.StaticHiveFunctionRegistry;
+import com.linkedin.coral.trino.rel2trino.functions.TrinoElementAtFunction;
+import com.linkedin.coral.trino.rel2trino.transfomers.MapStructAccessOperatorTransformer;
+import com.linkedin.coral.trino.rel2trino.transfomers.ToDateOperatorTransformer;
+
+import static com.linkedin.coral.trino.rel2trino.CoralTrinoConfigKeys.*;
+
+
+/**
+ * This utility class initialize a list of SqlCallTransformer which convert the function operators defined in SqlCalls
+ * from Coral to Trino on SqlNode layer
+ */
+public final class CoralToTrinoSqlCallTransformersUtil {
```

Comment:
I still think it's not suitable to call it `xxxUtil` and put it under `utils` package given it contains the key transformers. @wmoustafa let us know your opinion.

Reply:
Done.